### PR TITLE
Add the location to a user event

### PR DESCRIPTION
**Pull request by psoleckimoj**
Currently only displays the event info, add the location as well

- Add util function to create the title string
- Add tests

### CODE REVIEW

File: moj-node/server/utils/index.js [hunk 1]
```diff
@@ -39,10 +39,21 @@ function fixUrlForProduction(url, alternateUrl = config.hubEndpoint) {
   return url;
 }
 
+function getEventTitle(event) {
+  const fixedTitle = event.eventLocation
```
Comment (romidane) [inline moj-node/server/utils/index.js]: What do you mean by `fixedTitle`?

Comment (psoleckimoj) [reply moj-node/server/utils/index.js]: It's always going to be this title or this this title with a comment appended.

Comment (romidane) [reply moj-node/server/utils/index.js]: I think just calling it title will be fine here?

Comment (psoleckimoj) [reply moj-node/server/utils/index.js]: 👍

**Pull request closed without merging**